css/css-viewport/zoom/parsing/zoom-computed.html

```diff
@@ -5,12 +5,6 @@
 <script src="/resources/testharness.js"></script>
 <script src="/resources/testharnessreport.js"></script>
 <script src="/css/support/computed-testcommon.js"></script>
-<style>
-  #container {
-    container-type: inline-size;
-    width: 100px;
-  }
-</style>
```

Comment:
(This part is only relevant to the `cqw`-unit subtests, so I'm removing it here, since those tests are being split out by this commit.)